app/assets/stylesheets/base.scss.erb

```diff
@@ -434,14 +429,10 @@
 }
 
 .yourTopic .mapPerm:hover, .yourEdge .mapPerm:hover {
-  background-image: url(<%= asset_data_uri('arrowperms_sprite.png') %>);
-  background-position: -32px 0;
```

Comment:
nice, does this mean we can remove this file from the repo?

Reply:
no, we still use it for the synapse card

Reply:
actually wait, no I'll remove it, cause I removed it from synapse card too